src/spectrum.rs

```diff
@@ -0,0 +1,397 @@
+use std::fmt;
+use std::ops::{Add, Sub, Mul, Div};
+
+use IntoColor;
+use flt;
+use num::{Float, FromPrimitive, Zero};
+use ordered_float::OrderedFloat;
+use white_point::WhitePoint;
+use xyz::Xyz;
+
+/// The smallest wavelength represented in the `Spectrum`
+/// data structure. This is the first value in the
+/// map's wavelength.
+pub const MIN_LAMBDA: usize = 360;
```

Comment:
I was thinking that perhaps we should name these `{MIN, MAX}_WAVELENGTH`, since that's what they are, instead of which letter is used to represent them.